Library/src/Systems/ECommerce/ECommerce.cpp

```diff
@@ -113,6 +115,84 @@ void ProductInfoDtoToProductInfo(const chs_aggregation::ShopifyProductDto& Dto,
 	}
 }
 
+void CartDtoToCartInfo(const chs_aggregation::ShopifyCartDto& CartDto, csp::systems::CartInfo& Cart)
+{
+	if (CartDto.HasSpaceId())
+	{
+		Cart.SpaceId = CartDto.GetSpaceId();
+	}
+	else
+	{
+		FOUNDATION_LOG_ERROR_MSG("ShopifyCartDto missing SpaceId");
+	}
+
+	if (CartDto.HasShopifyCartId())
+	{
+		// There is a prefix added to the ID within the CHS system but it is not needed for our purposes.
```

Comment:
Please can we frame this in a way that is services-agnostic. Hopefully others will build services that can talk to CSP.
Something like, "We expect services may add a prefix to the ID, but it is not needed for our purposes". That will be more helpful

Reply:
I don't see this as services-agnostic. The bit we are stripping out is specific to Magnopus Services and leaves only the raw shopify cart. I think it would be good to update this comment to reflect that and make it clearer.